### ISSUE TITLE
Dark mode: Figma branch priority list

### ISSUE THREAD

**Issue by GCHQ-Developer-465** [mi6/ic-ui-kit#2441]
## Summary 
Primitive and Semantic tokens have been merged into main Figma branch. Work on new v3 DM branch, one component at a time and make changes to that branch. Once all components have been done, we merge this branch and v3 Figma is delivered.

## 💬 Description 
An exploratory branch for [ticket](https://github.com/mi6/ic-ui-kit/issues/2289) was created to capture all component tokens for DM work. This is almost at an end now. We need to start work on a fresh branch and perform the DM changes for real. Originally, we thought we could do this work one component at a time, merging into main - but this would affect how designers are currently working with ICDS in Figma. As such, we need to do this as one branch and merge when complete, thus delivering DM in one go.

**NOTE: When working on a component, please assign yourselves with "@" against a component. Once complete please tick the component off and move onto the next one and note down below that it is 'READY TO REVIEW' in the peer review section. Please refer to this [ticket](https://github.com/mi6/ic-ui-kit/issues/2289) for an example of this layout.**

This must be done first
- [ ] - Styles (not a component) @mi6-577 **READY TO REVIEW**
- [ ] - Atoms @mi6-577 **READY TO REVIEW**

Least dependent to most dependent.
**Least** 

These don't appear to have any nested instances
- [ ] - Skeleton @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Chip @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Classification Banner @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Divider @mi6-577 **READY TO REVIEW** (Colours aren't accessible) @GCHQDeveloper618 (ignoring colours)✅
- [ ] - Link @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Loading indicator @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅ 
- [ ] - Status tag @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Stepper @GCHQDeveloper618  **READY TO REVIEW** @mi6-577 ✅
- [ ] - Switch @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Tool tip @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Tree view @mi6-577 **READY TO REVIEW**

These definitely have a nested instance - possibly Atoms or one of the above
- [ ] - Accordion @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Back to top @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅ 
- [ ] - Badge @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Button @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Popover menu @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Search bar @mi6-577  @GCHQDeveloper618 (made changes) **READY TO RE-REVIEW**
- [ ] - Table of contents @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Text field @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Toggle button @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅

1 Component
- [ ] - Alert @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Breadcrumb @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Checkbox @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Data entity @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Date picker @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Drawer @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Navigation bar @mi6-577 **READY TO REVIEW** @GCHQDeveloper618✅
- [ ] - Radio button @mi6-577 **READY TO REVIEW** 
- [ ] - Select @mi6-577 **READY TO REVIEW** 
- [ ] - Side navigation @mi6-577 **READY TO REVIEW**
- [ ] - Tabs @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅

2 Components 
- [ ] - Empty state @mi6-577 **READY TO REVIEW** @GCHQDeveloper618 ✅
- [ ] - Pagination @mi6-577 **READY TO REVIEW**
- [ ] - Toast @GCHQDeveloper823 

3 Components 
- [ ] - Card horizontal @mi6-577 **READY TO REVIEW**
- [ ] - Card vertical @mi6-577 **READY TO REVIEW**
- [ ] - Footer @mi6-577 **READY TO REVIEW**
- [ ] - Top navigation @mi6-577 **READY TO REVIEW**

4 Components 
- [ ] - Hero @mi6-577 **READY TO REVIEW**

5 Components 
- [ ] - Dialog @mi6-577 **READY TO REVIEW**

7 Components 
- [ ] - Page header @mi6-577 **READY TO REVIEW**

7+ Components 
- [ ] - Data table @mi6-577 - to be completed once v3 merged into main

**Most**

**Comment by mi6-577** [comment]
Closing this ticket now as the v3 work has been published
